src/routes/migration/container/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { ReactElement, useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { isLocalStorageMigrated } from 'src/logic/currentSession/store/selectors'
+import setLocalStorageMigrated from 'src/logic/currentSession/store/actions/setLocalStorageMigrated'
+
+export type MigrationMessage = {
+  executeMigration: boolean
+  payload: string
+}
+
+// FIXME This is meant for easiness of testing purposes. Set to a fixed origin once ready
+const MESSAGE_ORIGIN = '*'
```

Comment:
I guess we can already replace this with the PR URL?

Reply:
Yes, we can, just left there because for testing locally was more convenient. Will change it just in case